catalog/core/catalog-core-solr-api-impl/src/main/java/ddf/catalog/source/solr/api/impl/AbstractSolrCollectionProvider.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.source.solr.api.impl;
+
+import com.google.common.collect.ImmutableList;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.source.solr.api.SolrCollectionProvider;
+import ddf.catalog.util.impl.DescribableImpl;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.codice.solr.factory.SolrCollectionConfiguration;
+import org.codice.solr.factory.SolrConfigurationData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class AbstractSolrCollectionProvider extends DescribableImpl
+    implements SolrCollectionProvider {
+  static final List<String> SOLR_CONFIG_FILES =
+      ImmutableList.copyOf(
```

Comment:
why do we need `copyOf`?  Can't we just use `of`?
`ImmutableList.of("dictionary.txt",
              "protwords.txt",
              "schema.xml",
              "solr.xml",
              "solrconfig.xml",
              "stopwords.txt",
              "stopwords_en.txt",
              "synonyms.txt")`